imperial_coldfront_plugin/views.py

```diff
@@ -51,3 +61,87 @@ def group_members_view(request: HttpRequest, user_pk: int) -> HttpResponse:
     group_members = GroupMembership.objects.filter(owner=user)
 
     return render(request, "group_members.html", {"group_members": group_members})
+
+
+@login_required
+def send_group_invite(request: HttpRequest) -> HttpResponse:
```

Comment:
This should be restricted to PIs, right?

I was able to generate an invite link when I wasn't the owner of a group which meant there was an error when going to that link because there was no group associated with that link. So that needs to be checked before the email is sent.

Reply:
Changed s.t. `send_group_invite` is forbidden unless user is a group owner.